content/en/tracing/service_catalog/setup.md

```diff
@@ -36,6 +39,24 @@ If the service, API, or custom library you are interested in is not listed on th
 
 To discover other services through existing Datadog telemetry such as infrastructure metrics, go to the **Discover Services** tab and follow instructions there. Discovery uses the `DD_SERVICE` [Unified Service Tagging convention][4] to locate services sending data to your Datadog organization.
 
+## Set up the GitHub Apps integration
+
+You can use the [GitHub Apps integration][5] to display links for service definitions in GitHub.
```

Comment:
```suggestion
Configure the [GitHub Apps integration][5] so that you can directly link from where you view the service's definition in Service Catalog to where it's stored and editable in GitHub.
```